AllHitAnalyze.py

```
@@ -1,234 +1,243 @@
+		x=0;
+		for ids in yLine:
+			ax1.plot(self.xAxis,ids,linestyle='--',alpha=1,linewidth=1,color=colors[x]);
+			x = (x+1)%7;
+
+		##正态分布函数曲线
+		if self.var > 0:
+    		pdfAxis = [PDFx(i) for i in self.xAxis]
+		else:
+    		pdfAxis = []
+		ax1.plot(self.xAxis,pdfAxis,linestyle='-',alpha=1,linewidth=1,color='grey',
+			label=f'Fitting all data\n(μ={self.avg}\n σ={self.std})');
```

Ruff (0.14.8)

String contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)